my.css

```diff
@@ -352,3 +352,166 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  max-width: 1150px;
+  width: 100%;
+  margin: 0 auto;
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 60%;
+  height: 844px;
+  
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 340px;
+  height: 400px;
+  padding-left: 10px;
+
+}
+.bags-img1 {
+  z-index: 1;
+  width: 540px;
+  height: 740px;
+  padding-left: 80px;
+}
+
+.bagclach__iner {
+  width: 45%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+
+}
+.card {
+  width: 480px;
+  padding: 40px 55px 45px 65px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)